### ISSUE TITLE
Warnings when building text containing literal blocks with i18n

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1057]
I get lots of warnings when I try to build text containing literal blocks, like this one:

```
some text::

   this is
   a literal block

more text
```

Warnings are similar to this one:

```
/path/to/file.rst:2: WARNING: Literal block expected; none found.
```

A test case is lp:ubuntu-packaging-guide with Spanish translation.

[Takayuki Shimizukawa](https://bitbucket.org/shimizukawa) has a [workaround](https://gist.github.com/4169008) for this, but I'm not sure if it can be applied in its current form.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1057
- Originally reported by: Dmitry Shachnev
- Originally created at: 2012-12-16T10:57:42.095

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-12-16 13:49:40+00:00_

Thanks for reporting! I start this problem correction.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-12-16 14:38:00+00:00_

Avoid unusable warning in using i18n translation. Closes #1057.
And there is related information at https://bitbucket.org/birkenfeld/sphinx/pull-request/86

→ <<cset 6e044aedbc9ba5a6ed83f099e2ad47786bfc46b4>>
